dask/utils.py

```diff
@@ -686,8 +686,10 @@ def unsupported_arguments(doc, args):
     return "\n".join(lines)
 
 
-def _derived_from(cls, method, ua_args=[], extra="", skipblocks=0):
+def _derived_from(cls, method, ua_args=None, extra="", skipblocks=0):
     """Helper function for derived_from to ease testing"""
+    ua_args = ua_args or []
```

Comment:
Noticed this while investigating https://github.com/dask/dask/pull/7873#discussion_r673944726.

Wasn't sure which one of the following to pick. Please let me know if one is preferred over the other.
```python
ua_args = ua_args or []
ua_args = [] if ua_args is None else ua_args
```

Reply:
What you have looks good to me.